suites/message/message_application.go

```diff
@@ -141,8 +141,20 @@ func MessageTest_MessageApplicationEdgecases(t *testing.T, factory state.Factori
 			exitcode.ErrIllegalArgument)
 	})
 
+	t.Run("receiver ID/Actor address does not exist", func(t *testing.T) {
+		td := builder.Build(t)
+		defer td.Complete()
+
+		alice, _ := td.NewAccountActor(drivers.SECP, aliceBal)
+
+		// Sending a message to non-existent actor must produce an error.
+		unknown := utils.NewIDAddr(t, 10000000)
+		td.ApplyFailure(
+			td.MessageProducer.Transfer(alice, unknown, chain.Value(transferAmnt), chain.Nonce(0)),
+			exitcode.SysErrInvalidReceiver)
+	})
```

Comment:
This test can be extended to also test sending to a non-existent _actor_ address (`utils.NewActorAddr`) in addition to the ID address tested here.